### DISCUSSION TITLE
Remap backspace on Colemak layout

### DISCUSSION THREAD

**Discussion by gekoke** [monkeytypegame/monkeytype#2711] [Questions] [answered]
It doesn't seem that emulating the Colemak layout remaps backspace to the Caps Lock key. Is there any way to do that strictly in the web app? Is there a technical limitation as to why this can't be done?

**Comment by Miodec** [comment, marked as answer]
Because its not meant to. Layout emulation just does a simple event character swap. If you want to backspace with your caps lock you would need to install it system wide I'm afraid.
